openmc/mgxs/mgxs.py

```diff
@@ -1977,9 +2031,9 @@ def get_slice(self, nuclides=[], in_groups=[], out_groups=[]):
         return slice_xs
 
     def get_xs(self, in_groups='all', out_groups='all',
```

Comment:
I've seen codes expect the moments to have been already multiplied by `(2l+1)/2` and others that do not.  OpenMC's MG mode does not expect the multiplication to have already occurred.  The docs should be updated in some way to let the user know what they're getting.  Hopefully they'd get very nonsensical results if they didn't know that, but 1) maybe not, and 2) might as well save them some time.